hive/runners/multi_agent_loop.py

```diff
@@ -104,22 +109,22 @@ def run_one_step(self, observation, turn, episode_metrics):
                 },
             )
         self._transition_info.update_all_rewards(reward)
-        return done, next_observation, turn
+        return terminated, truncated, next_observation, turn
 
-    def run_end_step(self, episode_metrics, done=True):
+    def run_end_step(self, episode_metrics, truncated=False):
         """Run the final step of an episode.
 
         After an episode ends, iterate through agents and update then with the final
         step in the episode.
 
         Args:
             episode_metrics (Metrics): Keeps track of metrics for current episode.
-            done (bool): Whether this step was terminal.
+            truncated (bool): Whether this step was terminal.
 
         """
         for agent in self._agents:
             if self._transition_info.is_started(agent):
-                info = self._transition_info.get_info(agent, done=done)
+                info = self._transition_info.get_info(agent, truncated=truncated)
```

Comment:
You need to send both truncated and terminated. Also this shouldn't run? I don't think you updated TransitionInfo with the new arguments.

Reply:
done.